docs/execution_providers/OpenVINO-ExecutionProvider.md

```diff
@@ -20,14 +20,15 @@ session.set_providers(['OpenVINOExecutionProvider'], [{Key1 : Value1, Key2 : Val
 *Note that this causes the InferenceSession to be re-initialized, which may cause model recompilation and hardware re-initialization*
 
 ### C/C++ API
-All the options (key-value pairs) need to be concantenated into a string as shown below and passed to OrtSessionOptionsAppendExecutionProviderEx_OpenVINO() API as shown below:-
+All the options shown below are passed to SessionOptionsAppendExecutionProvider_OpenVINO() API and populated in the struct OrtOpenVINOProviderOptions in an example shown below:-
 
 ```
-std::string settings_str;
-settings_str.append("Key1|Value1\n");
-settings_str.append("Key2|Value2\n");
-settings_str.append("Key3|Value3\n");
-Ort::ThrowOnError(OrtSessionOptionsAppendExecutionProviderEx_OpenVINO(sf, settings_str.c_str()));
+OrtOpenVINOProviderOptions options;
```

Comment:
Is the API to pass setting strings deprecated? or is it still active and the struct is just an alternative way to pass the options?

Reply:
Manohar there is just one API which works now and it takes only one device_type strings. It is given in ort_provider_factory.h 

std::string str("MYRIAD_FP16")
OrtSessionOptionsAppendExecutionProvider_OpenVINO(str)

Reply:
But the strings with the key values no longer work

Reply:
@smkarlap 
Yes I just verified from my end as well.
The API to pass settings_str will no longer work, it has been removed.
https://github.com/microsoft/onnxruntime/commit/ba739a80005dfd7eb1dbaac4d589d29c92393cfc

**Option 1:**
Yes, sahar says, one option is to pass only device_type strings

**Example:**
session_options.SetGraphOptimizationLevel(ORT_DISABLE_ALL);
std::string str("HETERO:MYRIAD,CPU");
Ort::ThrowOnError(OrtSessionOptionsAppendExecutionProvider_OpenVINO(session_options, str.c_str()));

**Option 2:**
The way how we pass other runtime options we have is using  OrtOpenVINOProviderOptions

**Example:**
sf.SetGraphOptimizationLevel(ORT_DISABLE_ALL);
      OrtOpenVINOProviderOptions ov_options;
      ov_options.device_type = "MULTI:MYRIAD,CPU";
      ov_options.num_of_threads = 11;
      ov_options.enable_vpu_fast_compile = true;
      sf.AppendExecutionProvider_OpenVINO(ov_options);

Reply:
@sfatimar @smkarlap .
Now that, sahar has already added OrtOpenVINOProviderOptions way of using c++ API.

**should we also add the c++ API option to just add device_type in our documentation?**

std::string str("HETERO:MYRIAD,CPU");
Ort::ThrowOnError(OrtSessionOptionsAppendExecutionProvider_OpenVINO(session_options, str.c_str()));

Reply:
Yeah..adding  the other API to pass just the device_type will be helpful.. however, we can list it as a deprecated API

Reply:
I have listed it as depreceated API in the documentation and updated